⚠️ CI failed ⚠️ - TPC-H dask query 1 NotImplemented Error & pytorch optuna ImportError

[Workflow Run URL](https://github.com/coiled/benchmarks/actions/runs/7454798588)

> FAILED tests/tpch/test_dask.py::test_query_1 - NotImplementedError: 'numeric_only=False' is not implemented in Dask.
FAILED tests/workflows/test_pytorch_optuna.py::test_hpo[client0] - ImportError: cannot import name 'TypeAliasType' from 'typing_extensions' (/opt/coiled/env/lib/python3.9/site-packages/typing_extensions.py)

Maybe @phofl is interested in the first one.

--- a/tests/workflows/test_pytorch_optuna.py
+++ b/tests/workflows/test_pytorch_optuna.py
@@ -26,9 +26,10 @@ optuna = pytest.importorskip("optuna")
             "torchvision",
         ],
         pip_options=[
-            "--force-reinstall",
             "--index-url",
             "https://download.pytorch.org/whl/cu118",
+            "--extra-index-url",
+            "https://pypi.org/simple",
         ],
         restart_workers=True,
     ),

